docs/features.rst

```diff
@@ -0,0 +1,44 @@
+Django Graph API features
+=========================
+
+This is a rough guide and not an exhaustive list. Please update this list as you work on adding new features.
+
+Supported
+---------
+
+Operations
+^^^^^^^^^^
+- queries
```

Comment:
It might be nice to give a brief description on this page of what queries, subscriptions, and mutations are.